--- a/include/target.mk
+++ b/include/target.mk
@@ -43,11 +43,14 @@ DEFAULT_PACKAGES.nas:=\
 	mdadm
 # For router targets
 DEFAULT_PACKAGES.router:=\
-	dnsmasq \
+	dnsmasq-full \
 	firewall \
 	ip6tables \
 	iptables \
+	ipset \
+	kmod-ipt-nat \
+	kmod-ipt-nat6 \
 	kmod-ipt-offload \
 	odhcp6c \
 	odhcpd-ipv6only \
 	ppp \
@@ -84,6 +84,22 @@ else
   endif
 endif
 
+ifneq ($(CONFIG_SMALL_FLASH),)
+DEFAULT_PACKAGES+=-coremark -htop -bash -openssh-sftp-server
+endif
+
+ifeq ($(ARCH),arm)
+DEFAULT_PACKAGES+=autocore-arm luci-app-cpufreq
+endif
+
+ifeq ($(ARCH),aarch64)
+DEFAULT_PACKAGES+=autocore-arm luci-app-cpufreq
+endif
+
+ifneq ($(CONFIG_USB_SUPPORT),)
+DEFAULT_PACKAGES+=automount
+endif
+
 # Add device specific packages (here below to allow device type set from subtarget)
 DEFAULT_PACKAGES += $(DEFAULT_PACKAGES.$(DEVICE_TYPE))
 
