--- a/package/feeds/kiddin9/uugamebooster/Makefile
+++ b/package/feeds/kiddin9/uugamebooster/Makefile
@@ -49,7 +49,7 @@ ifeq ($(ARCH),x86_64)
 	PKG_MD5SUM:=skip
 endif
 
-PKG_SOURCE_URL:=https://uu.gdl.netease.com/uuplugin/openwrt-$(UU_ARCH)/$(PKG_VERSION)/uu.tar.gz?
+PKG_SOURCE_URL:=https://github.com/tonyliangli/uugamebooster/releases/download/binaries/
 PKG_SOURCE:=$(PKG_NAME)-$(UU_ARCH)-$(PKG_VERSION).tar.gz
 
 STRIP:=true
